### PR TITLE
Move data_prep.rst from jdaviz to specutils as Working with Spectral …

### PR DESCRIPTION
**Pull request by PatrickOgle**
Add section to ReadTheDocs main index  that discusses how to use Spectral Cube to read in a cube, then collapse
to create a Spectrum1D.

### CODE REVIEW

**Review by eteq — APPROVED** [review]
Mostly looks good, but I added one suggestion to add some text giving a bit more context on how this is only one of the spectral cube formats, and we don't yet have a solution for the others.

File: docs/spectral_cube.rst
Comment (eteq) [inline docs/spectral_cube.rst:6]: I think the CI failure is due to this:
```suggestion
`SpectralCube <https://spectral-cube.readthedocs.io/en/latest/>`_ package then
```

Comment (PatrickOgle) [reply docs/spectral_cube.rst:6]: good catch!

**Review by javerbukh — APPROVED** [review]
Looks good. I would wait until tests are passing to merge, though. Any idea why they're failing?

File: docs/spectral_cube.rst
Comment (eteq) [inline docs/spectral_cube.rst:12]: oops I think the test failure is my bad - this should be either
```suggestion
``spectral_axis`` values - i.e., case 2 in
```
or
```suggestion
`~specutils.Spectrum1D.spectral_axis` values - i.e., case 2 in
```

**Comment by rosteen** [comment]
I'm pretty sure that (at least most of) the test failures are actually unrelated to this PR. Looks like test_loaders is hitting `ValueError: lookup_table should be an array with 1 dimensions.` running `gwcs_from_array`, I assume due to a change in an underlying package.

**Comment by eteq** [comment]
Yeah, agreed @rosteen, there's several unrelated failures.  But since this is only the docs, I've been looking specificially at the third build in the matrix (the one that has `build_docs` as its command - as an aside, #727 is about making this more readable!), you'll see there's documentation-related failures.  It looks like this change fixed the error I said above, but now there's a new one at line 30. 

Once that's passing I think we can merge this without worrying about the other failures right now.

**Comment by PatrickOgle** [comment]
@eteq Can this be fixed by an edit to the .rst file for this PR?

File: docs/spectral_cube.rst
Comment (PatrickOgle) [inline docs/spectral_cube.rst:31]:
```suggestion
load the data from the repository directly into a new ``SpectralCube``
```

File: docs/spectral_cube.rst
Comment (nmearl) [inline docs/spectral_cube.rst:57]:
```suggestion
    >>> sc_spec = sc.mean(axis=(1,2))
```

Just to offer an alternative that might make it conceptually easier for users, you can call the `mean` method directly which defaults to `projection=True` and avoid having to explain this edge case.

File: docs/spectral_cube.rst
Comment (nmearl) [inline docs/spectral_cube.rst:62]:
```suggestion
```

**Comment by javerbukh** [comment]
@PatrickOgle I think if you make the edits Nick suggested we can go ahead with merging this PR.

**Review by nmearl — APPROVED** [review]

File: docs/spectral_cube.rst
Comment (pllim) [inline docs/spectral_cube.rst:36]: For future reference, any doctest that downloads stuff needs to be marked as remote data. Otherwise, you'll hit the server too much and cause unrelated failures.

xref #838

**Pull request merged**
